python/test/ngraph/test_ops_reshape.py

```diff
@@ -110,6 +110,18 @@ def test_broadcast_v1():
     assert np.allclose(result, expected)
 
 
+@pytest.mark.skip_on_gpu
+def test_gather():
+    input_data = np.array([1.0, 1.1, 1.2, 2.0, 2.1, 2.2, 3.0, 3.1, 3.2], np.float32).reshape((3, 3))
+    input_indices = np.array([0, 2], np.int64).reshape(1, 2)
+    input_axes = np.array([1], np.int64)
+    result = run_op_node([input_data, input_indices, input_axes], ng.gather)
```

Comment:
Please test it also with `run_op_numeric_data` which doesn't convert inputs to nG Constants.

Reply:
Done